__init__.py

```diff
@@ -14,9 +14,11 @@
 
 import datetime
 import holidays
+import json
 import pytz
 import re
 import time
+from os.path import isfile
```

Comment:
I am trying to get us away from the os.path library and towards pathlib.

Reply:
done